### ISSUE TITLE
Is it possible to indicate orientation (head tracking) using Psmoveservice?

### ISSUE THREAD

**Issue by VRmaniaQ** [cboulay/PSMoveService#417]
Hello,
This is my partial quote from the Vridge forum:

> Hello,
Vridge is great. In connection with Psmoveserviece I have been using it since November. With Riftcat alone, I had no major problems. My play was to improve the quality of position tracking, the position of the hands and the perspective.
Conclusions: Beta version is great, because the startup fixes the exact position of the camera and the headset. No more initial calibration. Such precise positioning was previously impossible. Please do not change this :)
Questions: Can somehow manually specify how much FPS I would like to display on my phone? Currently, you can choose between 30 and 60. Suppose I want the number to be 45 or 50. Is it possible, in some simple way, for example, by editing a file? My question is directly related to the following (I'm thinking of setting the optimum working temperature for the magnetometer, reducing the amount of Fps).
Ahilles / weak point, mobile VR: In my opinion, the weak point of all this fun is the magnetometer. This sensitive to temperature changes, magnetic field fluctuations sensor introduces a lot of confusion. Even when calibrating the magnetometer in the "working area", when it reaches the operating temperature, the sensor can still indicate the wrong position (near the "N" direction).
I have read some scholarly work on this subject. I learned that the indications of these senators (magnetometer, accelerometer, gyroscope) should be improved programmatically. Because regardless of calibration attempts, I will always be inaccurate.
The question is: is it going to improve the quality of the magnetometer's display calibration even more than it has done so far in Vridge? Is there anything to do with it? Maybe it's also somewhere to edit? (I'm not a developer but simple editing will not be a problem) Ps3 move controllers have a magnetometer to determine the orientation also. Thanks to extensive calibration goes to calibrate it well enough. However, if the orientation of the head position fails, the proper orientations and positions of the controllers can be forgotten.
In short, the magnetometer on the phone is a problem, can it be corrected? I'm just wondering over the resignation of the telephone sensors, and determining the orientation using Psmoveservice using additional lighting points, I will try to ask if it is possible in their forum."

As I wrote at the end, is it possible to add 1 or 2 light points (total 2 or 3) to the headset? Of course changing the script in FreePie. So would head tracking and position tracking only work with Psmoveservice?

Greetings! :) 
VR maniaQ - youtube.

**Comment by VRmaniaQ** [comment]
Okey, I resolved the problem myself.
It turned out that the telephone magnetometer behaves identically to the ten of ps3move. When using the charger for the 4 controllers, charging them on on, connected to a PC. Their magnetometer magnetized, losing calibration, almost completely crazy.
Following this track, I began to wonder if the phone that was charging when plugged in via USB thatering also did not behave like ps3move.
Bingo! I tried to test the Wifi connection. I calibrated the magnetometer again on my phone and found that there was a big improvement. To completely reset the settings and magnetizing the magnetometer, I removed the battery for 20 minutes. Putting it away from the phone. After turning on the phone and re-calibrating, everything started to work perfectly. The position at the end works as it should.
Btw. I tried a half year ago Wifi at a friend ... the quality was unacceptable. Delay was cosmic.
Since then, I can not imagine playing through USB. The delay is so negligible that it hardly discern any difference.
Riftcat! ! ! You are brilliant! ! ! VR without cable is 200% better than before. Amazing freedom of movement, regardless of rotation.
This is the VR I was waiting for. Thanks to Riftcat! ! ! ! :)
More about my settings, patents, explained on the google + group in the subject: "Flawless tracking." Without shifts, mistakes, 360 degrees. Four cameras. " 4th of April.
I'm going to update some new patents and new knowledge soon.
Greetings to all ! :) VR maniaQ - youtube. https://www.youtube.com/channel/UCLrhuKIo2_yfAGamH7xYCYA

**Comment by VRmaniaQ** [comment]
Something like that I mean. Funny, HTC did what I had in mind. Or something like that. So a double or triple marker.

https://we.tl/EPEfLBqYAS
